### ISSUE TITLE
DevelopmentCodeFragment does not work in Files with namespace

### ISSUE THREAD

**Issue by Stoffo** [phpmd/phpmd#532]
Hi,

the DevelopmentCodeFragment Rule does not detect unwanted functions like `var_dump`, `print_r` when they occur in a non-global namespace.

### This works:
```php
<?php

class Foo
{
    private function baz()
    {
        var_dump("");
    }
}
```

`The method Foo::baz() calls the typical debug function var_dump() which is mostly only used during development.
`

### This does not work:
```php
<?php

namespace baz;

class Foo
{
    private function baz()
    {
        var_dump("");
    }
}
```

**Comment by mstaack** [comment]
any updates on this one?

**Comment by jframos0** [comment]
it's still not working with namespace :(
